lib/contestant.rb

```diff
@@ -0,0 +1,25 @@
+class Contestant
+
+  attr_reader :full_name,
+              :age,
+              :state_of_residence,
+              :spending_money,
+              :game_interests
+
+  def initialize(contestant_info)
+    @full_name = contestant_info[:first_name] + " " + contestant_info[:last_name]
+    @age = contestant_info[:age]
+    @state_of_residence = contestant_info[:state_of_residence]
+    @spending_money = contestant_info[:spending_money]
+    @out_of_state = false
```

Comment:
this doesn't quite work for `@out_of_state`-- you needed to check whether the `@state_of_residence` was equal to "CO" -- with what you have written, you have no way for a contestant to ever be out of state. This would be better suited as a method, rather than an attribute. 

```
  def out_of_state?
    @state_of_residence != "CO"
  end
```